### PR TITLE
Document internal structure of the Stores App (fixes #17)

### LINKED ISSUE #17: Document internal structure of the Stores App

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by flodolo** [comment]
This is blocked by #42, since I've started on top of it.

**Comment by flodolo** [comment]
This one is ready for review, asking @TheoChevalier since he's the only one with some knowledge of the code.

File: tools/stores_l10n/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+{$_('Tracker')}
+
+{$_('From Mozilla<br>A brand you trust')}
+OUT;
+};
+```
+
+Similarly to a Django template, each string is wrapped in a translation function `$_('')` (defined in [app/inc/utilities.php](https://github.com/mozilla-l10n/stores_l10n/blob/master/app/inc/utilities.php#L5)).
+Strings are grouped into a variable, like `$screenshots` in this case: this allows, for example, to determine the overall length of all strings included in a section, to see if it exceeds the [limits imposed by stores](https://github.com/mozilla-l10n/stores_l10n/blob/master/app/inc/constants.php#L15-L22).
```
Comment (TheoChevalier) [inline tools/stores_l10n/README.md]: Maybe worth mentioning that if translations exceed the max length, they are considered "not done" until localizers fix them (unless this has changed), and that length errors are not displayed in the webdashboard.

Comment (flodolo) [reply tools/stores_l10n/README.md]: Good point. Added note.

**Review by TheoChevalier — APPROVED** [review]
LGTM :)

**Comment by flodolo** [comment]
Thanks! 👍

**Pull request merged**